opslevel/datasource_opslevel_property_definition.go

```diff
@@ -72,7 +72,7 @@ func (d *PropertyDefinitionDataSource) Schema(ctx context.Context, req datasourc
 				Required:            true,
 			},
 			"id": schema.StringAttribute{
-				MarkdownDescription: "Terraform specific identifier.",
+				MarkdownDescription: "The ID of this resource.",
```

Comment:
This description will eventually apply to all or most `id` fields in schemas.
Pulled this description from [the registry](https://registry.terraform.io/providers/OpsLevel/opslevel/latest/docs/data-sources/property_definition), also elsewhere in repo

Reply:
@davidbloss Thoughts on using a `validator` for all `id` fields?

Reply:
It would not help us with computed/returned values like we have in datasources.
A `validator` for `id` fields would be really helpful for managed resources since an `id` would be provided by a customer.

Reply:
If a field is computed there is no need for a validator - you only put validators on fields a customer supplies to ensure their config data is valid at `plan` time rather then having to make an API call at `apply` time.  Customers should NOT be supplying us ID for `resource` types and only should ever supply us an ID in the `identifier` field of a datasource (or maybe other config fields in weird cases like we saw for property).